### DISCUSSION TITLE
How do I compile a windows installation package?

### DISCUSSION THREAD

**Discussion by GanZhiXiong** [KRTirtho/spotube#1767] [Q&A]
1. What is the difference between the two files windows/packaging/exe/inno_setup.iss and scripts/windows-setup-creator.iss?
2. There are innoinstall, inno, and choco in the Makefile. These configurations should be related to Windows. I don't know the difference between them and don't know which command to use.

**Comment by KRTirtho** [comment]
You'll need InnoSetup installed and the inno CLI `ISCC.exe` excessible in the terminal.  Make sure Pub-Cache is also in ENV path.
Then, you can run:
```shell
$ dart pub global activate flutter_distributor
$ flutter_distributor package --platform=windows --targets=exe --skip-clean
```

**Reply by GanZhiXiong** [reply]
Thank you very much for your reply, I will try it.
This answer was beyond my expectation, I didn't think the command would be like this, thank you!
